recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
-{% set major_minor = "2.52" %}
-{% set version = major_minor + ".1" %}
-{% set sha256 = "dc19d20fb6b24d6b6da7cbe1b2190b38ae6ad64e6b93fecdcce71b995f43c975" %}
+{% set major_minor = "2.62" %}
+{% set version = major_minor + ".0" %}
```

Comment:
can you please flip the logic here so that major_minor is computed from the version number, so that the bot will hopefully do the correct update in the future

Reply:
I still need to figure out exactly what is going on with versions. First off, they use a weird versioning scheme where the even numbered version (second number) is stable and the odd ones are unstable (ref: https://www.gtkmm.org/en/download.html). From what I can tell, it's best to avoid the odd numbered versions (for instance, I'm not using 2.63.1 of this package because there's a scary "distro packages should probably not package this yet" in the [NEWS](https://github.com/GNOME/glibmm/blob/a75552366355ffa8ab850921ceef546e36705266/NEWS#L2) file). 

So the bot would need to know about that to be useful. They also seem to do a lot of backports, based on the dates at https://ftp.gnome.org/pub/GNOME/sources/glibmm/

Secondly, different versions have different version requirements for dependencies. It would probably help to have the pinning machinery here, or somehow extract the dependency metadata into the meta.yaml. The many backports also make me worried about this.

Ultimately I'm trying to build inkscape, which requires a bunch of gtk stuff. I'm hoping I can avoid having to build gtk3, but I don't know yet if that is possible. But some of the other libraries depend on gtk2 or gtk3, depending on the version. For instance, gtkmm, which depends on this and I am working on building, switched from gtk2 to gtk3 in version 2.90 https://github.com/GNOME/gtkmm/blob/248c85a2ddc4f51a7a4b806e2a734c2545737763/NEWS#L3848. 

So the whole thing is a mess and I don't actually know yet if I'll be able to get a handle on all of it. If any of the conda-forge tooling can help me with this, let me know, or if there's someone who knows more about the GNOME libraries who can help. For now, I think I might try to get things built locally before I try to get things updated on conda-forge. 

Also I will probably only work on this on and off.